  export const blockUser = async () => {
    try {
        
    } catch (error) {
        
    }
  }